### PR TITLE
Add information about returned values for some hooks [DOCS]

### PR DESCRIPTION
**Pull request by scarletfog**
### Context
For some hooks, we can describe what does return value do.

### Types of changes

- [x] Improvement in the documentation.

### Related issue(s):
1. https://github.com/handsontable/docs/issues/156
2. https://github.com/handsontable/docs/issues/160
3. https://github.com/handsontable/docs/issues/163
4. https://github.com/handsontable/docs/issues/165
5. https://github.com/handsontable/docs/issues/167
6. https://github.com/handsontable/docs/issues/168
7. https://github.com/handsontable/docs/issues/169
8. https://github.com/handsontable/docs/issues/170
9. https://github.com/handsontable/docs/issues/171

@pnowak can we say those are "actions"? I wanted to find a pattern we can re-use in sentences.

### CODE REVIEW

**Review by pnowak — APPROVED** [review]
👌

File: src/pluginHooks.js [hunk 1]
```diff
@@ -648,6 +649,7 @@ const REGISTERED_HOOKS = [
    * @param {CellCoords} start Object containing information about first filled cell: `{row: 2, col: 0}`.
    * @param {CellCoords} end Object containing information about last filled cell: `{row: 4, col: 1}`.
    * @param {Array[]} data 2D array containing information about fill pattern: `[["1", "Ted"], ["1", "John"]]`.
+   * @returns {*|boolean} If false, then the the autofill action will be canceled.
```
Comment (AMBudnik) [inline src/pluginHooks.js:652]: then the the autofill - 2 x 'the'

**Review by AMBudnik — COMMENTED** [review]
As far as I remember we avoid using the future tense. Maybe we can use something general for all the actions, like
> If false is returned the default action is canceled.

**Comment by scarletfog** [comment]
@AMBudnik suggestions applied, you can check again!

**Review by AMBudnik — APPROVED** [review]
Approved :)

**Pull request merged**